metadata.json

```diff
@@ -0,0 +1,38 @@
+{
+    "id": "LandsMC,
+    "name": "Lands",
+    "description": "O servidor destinado a ser o melhor!",
+    "addresses": [
+        "Landsmc.com",
+        "Stydismc.com"
+    ],
+    "primaryAddress": "landsmc.com",
+    "minecraftVersions": [
+        "1.7.*",
+        "1.8.*"
+    ],
+    "primaryMinecraftVersion": "1.8.9",
+    "primaryColor": "#0da1de",
+    "secondaryColor": "#081723",
+    "primaryLanguage": "pt-br",
+    "languages": [
+        "pt-br",
+        "pt-pt"
+    ],

```

Comment:
For languages that include a `-`, the correct way to put this is to have the last end be fully uppercase. In your case: pt-BR and pt-PT